malicious/npm/3a-look/MAL-0000-3a-look.json

```diff
@@ -0,0 +1,53 @@
+{
+  "modified": "2023-08-21T20:12:58Z",
+  "published": "2023-04-27T00:00:00Z",
+  "schema_version": "1.5.0",
+  "summary": "Malicious code in 3a-look (npm)",
+  "details": "Lazarus Group targeting blockchain and cryptocurrency companies by exploiting software supply chains through malicious npm packages and social engineering tactics",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "npm",
+        "name": "3a-look"
+      },
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0"
+            }
+          ]
+        }
+      ],
```

Comment:
Just checking if this is intentional.

If you have specific versions that are vulnerable using the `versions` key is probably better.

If the entire package is malicious then this is acceptable.